verror/index.d.ts

```diff
@@ -1,12 +1,12 @@
-// Type definitions for verror v1.9.0
+// Type definitions for verror v1.9.1
```

Comment:
That is, if someone made a mistake in the definitions, we have to wait for the next release to get it fixed?